index.html

```diff
@@ -43,7 +43,22 @@ <h3>Backgrounds</h3>
       </section>
     </div>
     <div class="column">
-      <section class="bear-portrait">
+      <section class="bear-portrait color-background">
+        <img src="bab-assets/bear.png" alt="bear">
```

Comment:
Hide classes were added here to the images so that once we add the properties of the hide class, all images located in the display div box are hidden until we began work on properly position each individual one.  We also added class names according to category so that we could target each one separately when the time comes.